spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/data/couchbase/CouchbaseReactiveRepositoriesAutoConfigurationTests.java

```diff
@@ -82,7 +81,7 @@ public void noRepositoryAvailable() throws Exception {
 	public void doesNotTriggerDefaultRepositoryDetectionIfCustomized() {
 		load(CouchbaseReactiveRepositoriesAutoConfigurationTests.CustomizedConfiguration.class);
 		assertThat(this.context.getBeansOfType(ReactiveCityCouchbaseRepository.class))
-				.isEmpty();
+				.isNotNull();
```

Comment:
That assertion on something that returns a `Map` is wrong (it will always be non null). The purpose of this test is to check that the auto-configuration backs off as soon as there is a custom directive for the repositories.